extensions/anonaddy/.eslintrc.json

```diff
@@ -5,6 +5,40 @@
     "node": true
   },
   "parser": "@typescript-eslint/parser",
-  "plugins": ["@typescript-eslint"],
-  "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended", "prettier"]
+  "plugins": ["import-x", "react"],
+  "extends": [
+    "@raycast",
+    "plugin:import-x/recommended",
+    "plugin:react/recommended",
+    "plugin:react-hooks/recommended-legacy",
+    "prettier"
+  ],
+  "rules": {
+    "@typescript-eslint/consistent-type-definitions": ["error", "type"],
+    "@typescript-eslint/consistent-type-imports": "error",
+    "arrow-body-style": ["error", "as-needed"],
+    "import-x/no-unresolved": "off",
+    "import-x/order": [
+      "error",
+      {
+        "alphabetize": {
+          "caseInsensitive": true,
+          "order": "asc"
+        },
+        "groups": [["builtin", "external"], "internal", ["parent", "sibling"], "type"],
+        "newlines-between": "always"
+      }
+    ],
+    "no-console": "warn",
+    "react/react-in-jsx-scope": "off",
+    "react/jsx-sort-props": ["error", { "callbacksLast": true, "reservedFirst": true, "shorthandFirst": true }],
+    "react/self-closing-comp": "error",
+    "sort-imports": ["error", { "ignoreDeclarationSort": true }],
+    "sort-keys": "error"
+  },
+  "settings": {
+    "react": {
+      "version": "detect"
+    }
+  }
```

Comment:
Please don't change the ELint settings to your personal preference. We should keep all extensions' dev experiences consistent.

Reply:
How should this look like? Just extending `@raycast`? I was following the advise of https://developers.raycast.com/information/developer-tools/eslint#customization

Reply:
Just extending `@raycast` should be fine. Like this: https://github.com/raycast/extensions/blob/main/extensions/say/.eslintrc.json.